lit_llama/adapter.py

```diff
@@ -69,7 +69,7 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
             self.rope_cache = build_rope_cache(
                 seq_len=self.block_size,
                 n_elem=self.n_embd // self.n_head, 
-                dtype=self.c_attn.weight.dtype,
```

Comment:
Since the attention layer is quantized, dtype will be int8 here, but rope cache must be constructed with float dtype. In model.py we already made that fix, carrying it over here too.